test/serializers/attribute_test.rb

```diff
@@ -95,6 +95,29 @@ def test_virtual_attribute_block
 
         assert_equal(expected, hash)
       end
+
+      def test_conditional_attributes
+        serializer = Class.new(ActiveModel::Serializer) do
+          attribute :if_attribute_included, if: :true
+          attribute :if_attribute_excluded, if: :false
+          attribute :unless_attribute_included, unless: :false
+          attribute :unless_attribute_excluded, unless: :true
+
+          def true
+            true
+          end
+
+          def false
+            false
+          end
+        end
+
+        model = ::Model.new
+        hash = serializable(model, serializer: serializer).serializable_hash
+        expected = { if_attribute_included: nil, unless_attribute_included: nil }
+
+        assert_equal(expected, hash)
```

Comment:
we should look into if rubocop has a linter for parens around assert.. hashtag lazy comment